### PR TITLE
Some logs and comments fix the scheduler misspelled

### PR DESCRIPTION
**Pull request by wgliang**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: test/integration/scheduler_perf/scheduler_test.go [hunk 1]
```diff
@@ -144,7 +144,7 @@ func schedulePods(config *testConfig) int32 {
 			glog.Fatalf("%v", err)
 		}
 		// 30,000 pods -> wait till @ least 300 are scheduled to start measuring.
-		// TODO Find out why sometimes there may be scheduling blips in the beggining.
+		// TODO Find out why sometimes there may be scheduling blips in the begginning.
```
Comment (nikhita) [inline test/integration/scheduler_perf/scheduler_test.go]: s/begginning/beginning.

Comment (nikhita) [reply test/integration/scheduler_perf/scheduler_test.go]: http://www.thesaurus.com/browse/beginning

Comment (wgliang) [reply test/integration/scheduler_perf/scheduler_test.go]: It's my fault,thanks!

**Comment by wgliang** [comment]
This PR‘s changes has been included in the #59464, I will close.

**Pull request closed without merging**